src/Bithumb/Spot/API/Candle.jl

```diff
@@ -0,0 +1,107 @@
+module Candle
+
+export CandleQuery,
+    CandleData,
+    candle
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Bithumb
+using CryptoAPIs.Bithumb: Data
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum TimeInterval m1 m3 m5 m10 m30 h1 h6 h12 h24
+
+Base.@kwdef struct CandleQuery <: BithumbPublicQuery
+    order_currency::String
+    payment_currency::String
+    interval::TimeInterval
+end
+
+function Serde.ser_type(::Type{<:CandleQuery}, x::TimeInterval)::String
+    x == m1  && return "1m"
+    x == m3  && return "3m"
+    x == m5  && return "5m"
+    x == m10 && return "10m"
+    x == m30 && return "30m"
+    x == h1  && return "1h"
+    x == h6  && return "6h"
+    x == h12 && return "12h"
+    x == h24  && return "24h"
+end
+
+Serde.SerQuery.ser_ignore_field(::Type{CandleQuery}, ::Val{:order_currency}) = true
+
+Serde.SerQuery.ser_ignore_field(::Type{CandleQuery}, ::Val{:payment_currency}) = true
+
+Serde.SerQuery.ser_ignore_field(::Type{CandleQuery}, ::Val{:interval}) = true
```

Comment:
```suggestion
Serde.SerQuery.ser_ignore_field(::Type{CandleQuery}, ::Val{:order_currency}) = true
Serde.SerQuery.ser_ignore_field(::Type{CandleQuery}, ::Val{:payment_currency}) = true
Serde.SerQuery.ser_ignore_field(::Type{CandleQuery}, ::Val{:interval}) = true
```
remove line breaks between methods